nodemanagement/node_management_worker.go

```diff
@@ -360,3 +361,16 @@ func (n *NodeManagementWorker) UpdateStatus(policyName string, status *exchangec
 func nmwlog(message string) string {
 	return fmt.Sprintf("Node management worker: %v", message)
 }
+
+// This will remove nmps and statuses from the local db and the exchange
+func (n *NodeManagementWorker) HandleUnregister() {
+	if err := persistence.DeleteAllNodeManagementPolicies(n.db); err != nil {
+		glog.Errorf(nmwlog(fmt.Sprintf("Error removing node management policies from the local db: %v", err)))
+	}
+	if err := persistence.DeleteAllNMPStatuses(n.db); err != nil {
+		glog.Errorf(nmwlog(fmt.Sprintf("Error removing node management policy statuses from the local db: %v", err)))
+	}
+	if err := exchange.DeleteNodeManagementAllStatuses(n, exchange.GetOrg(n.GetExchangeId()), exchange.GetId(n.GetExchangeId())); err != nil {
```

Comment:
This might have timing issue because the node might have been removed by https://github.com/open-horizon/anax/blob/master/governance/shutdown.go#L103. Please move this to shutdown.go.